app/src/main/java/org/wikipedia/main/MainDrawerView.java

```diff
@@ -71,10 +71,7 @@ public void updateState() {
             accountAvatar.setVisibility(View.VISIBLE);
             accountWikiGlobe.setVisibility(View.GONE);
             notificationsContainer.setVisibility(View.VISIBLE);
-            if (Prefs.isSuggestedEditsAddDescriptionsUnlocked()
-                    || Prefs.isSuggestedEditsAddCaptionsUnlocked()) {
-                editTasksContainer.setVisibility(VISIBLE);
-            }
+            editTasksContainer.setVisibility(VISIBLE);
```

Comment:
Can you add a TODO as a reminder for us to remove it after moving to the navigation menu?

Reply:
This is handled in #625